### ISSUE TITLE
Types do not exist

### ISSUE THREAD

**Issue by littleBitsman** [DevAX1T/rbxcloud#6]
I'll probably be able to make these but it will take a while

Issue: TypeScript hates the module due to no typings

**Comment by littleBitsman** [comment]
@DevAX1T

**Comment by DevAX1T** [comment]
Yeah I'm rewriting this module in TypeScript, but I've been quite busy with life and procrastination

**Comment by littleBitsman** [comment]
> Yeah I'm rewriting this module in TypeScript, but I've been quite busy with life and procrastination

Alright gl with that then
